functests/test_resources/door_entry_story.yaml

```diff
@@ -8,25 +8,29 @@
 # This story creates a number of doors and access cards. An 'open door' event is then 
 # created with attached photo. Finally all doors and cards are listed and all events on 
 # the the Courts of Justice Front Door and all events on the Access Card 2 are listed.
+# #
+# For demo purposes all display names and asset names are appended with a date to
+# ensure that assets are unique. Change this value if you want to recreate the assets.
 steps:
   - step:
       action: ASSETS_CREATE_IF_NOT_EXISTS
       description: Create a door in Paris at Jitsuin offices.
-    signature:
-      attributes:
-        arc_display_name: Jitsuin Paris Front Door
+    selector:
+      - attributes.arc_display_name
     behaviours:
       - RecordEvidence
       - Attachments
     attributes:
+      arc_display_name: Jitsuin Paris Front Door 2022-03-04
```

Comment:
Probably didn't mean to commit this funky name. There are a bunch of these I won't point them all out.

Reply:
these have been replaced with getting a namespace from environment in the estate info PR